src/SpaceStationType.cpp

```diff
@@ -247,7 +247,7 @@ bool SpaceStationType::GetShipApproachWaypoints(const unsigned int port, DockSta
 	if (pPort) {
 		TMapBayIDMat::const_iterator stageDataIt = pPort->m_approach.find(stage);
 		if (stageDataIt != pPort->m_approach.end()) {
-			const matrix4x4f &mt = pPort->m_approach.at(stage);
+			const matrix4x4d &mt = pPort->m_approach.at(stage);
```

Comment:
If you're concerned about post-transform normality of the vector, this should instead be:

```c++
matrix4x4d mt = matrix4x4d(pPort->m_approach.at(stage));
mt.Renormalize();
...
```